documentation/src/main/asciidoc/topics/config_examples/hotrod-client.properties

```diff
@@ -0,0 +1,19 @@
+# Hot Rod client configuration
+infinispan.client.hotrod.server_list = 127.0.0.1:11222
+infinispan.client.hotrod.marshaller = org.infinispan.jboss.marshalling.commons.GenericJBossMarshaller
```

Comment:
Change the value to `org.infinispan.commons.marshall.ProtoStreamMarshaller` as the GenericJbossMarshaller soon won't be available without adding an additional jar.